BossMod/BossModule/ArenaBounds.cs

```diff
@@ -209,4 +209,69 @@ public override WDir ClampToBounds(WDir offset, float scale)
             return offset;
         }
     }
+
+    public class ArenaBoundsTri : ArenaBounds
+    {
+        private const float sqrt3 = 1.73205080757f; // Square root of 3
+
+        public ArenaBoundsTri(WPos center, float sideLength) 
+            : base(center, sideLength * sqrt3 / 3) { } // HalfSize is the radius of the circumscribed circle
+
+        public override IEnumerable<WPos> BuildClipPoly(float offset = 0)
+        {
+            // Calculate the vertices of the equilateral triangle
+            var height = HalfSize * sqrt3; // Height of the equilateral triangle
+            var halfSide = HalfSize;
+            yield return Center + new WDir(-halfSide, height / 3);
+            yield return Center + new WDir(halfSide, height / 3);
+            yield return Center + new WDir(0, -2 * height / 3);
+        }
+
+        public override Pathfinding.Map BuildMap(float resolution = 0.5f)
+        {
+            // BuildMap implementation for equilateral triangle
+            // This is a simplified example and would need to be adapted based on specific pathfinding requirements
+            throw new NotImplementedException();
```

Comment:
Should be very similar to circle - create a map and block out stuff outside triangle. Keeping it like that and throwing an exception will cause issues (eg with replay viewer).